testhelper/testhelper.go

```diff
@@ -87,6 +87,11 @@ func getStubLidar(t *testing.T) *inject.Camera {
 		t.Error("TEST FAILED stub lidar Projector called")
 		return nil, transform.NewNoIntrinsicsError("")
 	}
+	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
+		return camera.Properties{
+			SupportsPCD: true,
+		}, nil
+	}
```

Comment:
do we need this here? or should we implement it with a t.Error similar to the functions above?

Reply:
Similarly to before, we need this camera to support the properties as we are always checking them - in the functions above, we are returning an error if they are called, since we want to make sure we are using offline data. However, since the properties function is more of a sanity check, we do need this to be properly implemented